docs/experiments-microblog.md

```diff
@@ -0,0 +1,44 @@
+# Anserini Experiments on TREC Microblog Collection
+
+
+### Tweets2011 collection
+
+Note that the Tweets2011 collection is distributed as a list of tweet ids that you have to download yourself, so the 
+effectiveness results you'll get should be similar, but will likely not be identical to the scores reported here.
```

Comment:
Because the tweets might have been deleted?

To be pedantic, you mightn't get the same scores regardless. It depends on docid assignment, and how you break ties in ranking. I'm not entirely sure how lucene does either of these, so feel free to ignore/comment back.